MusicTaste/MusicTaste/SceneDelegate.m

```diff
@@ -15,9 +9,11 @@ @implementation SceneDelegate
 
 
 - (void)scene:(UIScene *)scene willConnectToSession:(UISceneSession *)session options:(UISceneConnectionOptions *)connectionOptions {
-    // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
-    // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
-    // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
+    if (PFUser.currentUser){
```

Comment:
If current user is already active you dont have to sign in again and will send you straight to the connect View